test/CMakeLists.txt

```diff
@@ -205,6 +205,15 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
     ENVIRONMENT COVERAGE_FILE=.coverage.cover_tests
     COMMAND ${PYTHON_EXECUTABLE} ${pytest} ./test/cover
     SCRIPT_PATTERNS test/cover/*.py test/cover/*.json)
+  nrn_add_test_group(NAME example_nmodl MODFILE_PATTERNS share/examples/nrniv/nmodl/*.mod
+                                                         share/examples/nrniv/nmodl/*.inc)
+  nrn_add_test(
+    GROUP example_nmodl
+    NAME example_nmodl
+    COMMAND
+      bash -ceo pipefail
+      [=[ pushd share/examples/nrniv/nmodl && find . -type f -name "*.hoc" | xargs -I{} special {} ]=]
```

Comment:
`nitpick`: a pattern like below doesn't suit here?

```cmake
file(GLOB files "*.hoc")
foreach(file ${files})
  ...
  nrn_add_test(....)
endforeach()
```

Reply:
Yeah, that would work, but that would end up polluting our ctest logs? Since we just make sure the scripts run and we have no error (no real coverage).

Reply:
i.e. one entry for each `nrn_add_test`